### ISSUE TITLE
MultiSelect: 'Scrollable region must have keyboard access' accessibility violation

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#512]
Install the Axe Dev Tools for Accessibility plug-in.
Navigate to https://v17.primeng.org/multiselect on Chrome browser
Scroll to Template section and open the multiselect control
Scan the page with Axe

Observe 'Scrollable region must have keyboard access' accessibility violation

![Image](https://github.com/user-attachments/assets/c5562dca-531a-4038-8a5a-22d27c0971d4)

Angular version: 18.2.9
PrimeNG version: v17
Node version: 18.19.1

> Migrated from `primefaces/primeng#18184` — originally by `@elena-ung` on 2025-04-28

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-10-26T00:20:52.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
